go.mod

```diff
@@ -3,11 +3,13 @@ module github.com/integr8ly/application-monitoring-operator
 go 1.13
 
 require (
```

Comment:
i don't see any new imports in modified src files. `go.mod` & `go.sum` may need to be regenerated or reverted